### PR TITLE
feat(documentation): implement new docs styles and webfonts

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

File: packages/documentation/.storybook/styles/components/fonts.scss
Comment (gfellerph) [inline packages/documentation/.storybook/styles/components/fonts.scss:1]: It really comes down to personal preference and I don't really want to spend more time on this, but I also want a crisp docs page. Can we try to use Fira Code as a font (for body text as well)? https://fonts.google.com/specimen/Fira+Code/tester?query=fira&noto.query=plex

image

File: packages/documentation/.storybook/styles/components/typo.scss
Comment (gfellerph) [inline packages/documentation/.storybook/styles/components/typo.scss:1]: https://preview-3161--swisspost-design-system-next.netlify.app/?path=/docs/677cfabf-dbf0-4de2-ad07-6d5bfb9e2375--docs#elements shows headings in monospace. I think they should show up in frutiger.

File: packages/components/src/components/post-tabs/readme.md [hunk 1]
```diff
@@ -47,6 +47,14 @@ Type: `Promise<void>`
 | `"tabs"`    | Slot for placing tab headers. Each tab header should be a <post-tab-header> element. |
 
 
+## Shadow Parts
+
+| Part        | Description |
+| ----------- | ----------- |
+| `"content"` |             |
+| `"tabs"`    |             |
```
Comment (gfellerph) [inline packages/components/src/components/post-tabs/readme.md:55]: Would be nice to have a description here and maybe in a separate ticket an explanation on how to style web component parts.

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/documentation/.storybook/styles/components/fonts.scss
Comment (oliverschuerch) [reply packages/documentation/.storybook/styles/components/fonts.scss:1]: I've added the "FiraCode" (@gfellerph) and "InterDisplay" (@oliverschuerch ) webfonts, but I didn't remove "Reddit Sans" (@rouvenpost) so far, to be able to compare them side-by-side.

1. Open two browser windows and load the preview url: https://preview-3161--swisspost-design-system-next.netlify.app/.
2. Open the dev tools in one of the windows and search after `--post-docs-font-sansserif`, to remove edit the value and therefore switch to a different font.
😉

File: packages/components/src/components/post-tabs/readme.md (hunk 1 above)
Comment (oliverschuerch) [reply packages/components/src/components/post-tabs/readme.md]: There you go: #3171

**Comment by oliverschuerch** [comment]
We've decided to go with "FiraCode" and "Reddit Mono".
I will remove the ohter fonts and update the PR.

**Review by alizedebray — APPROVED** [review]
Nice! I've seen some little things: 
- The control table is truncated at the bottom: 
image
- Alerts within the control table are quite big:
image
- It would be good to round the tabs on hover as well:
image

**Pull request merged**